src/nginxconfig/util/logging.js

```diff
@@ -25,7 +25,7 @@ THE SOFTWARE.
 */
 
 export const accessLogPathDefault = '/var/log/nginx/access.log';
-export const accessLogParamsDefault = 'buffer=512k flush=1m';
+export const accessLogParamsDefault = 'combined buffer=512k flush=1m';
```

Comment:
Instead of adding it here, I think you want to add it to the `getDomainAccessLog` logic below, when the Cloudflare log format is not enabled?

Otherwise, you'll end up specifying two log formats when Cloudflare logging is enabled?

Reply:
@MattIPv4 I think you are right. I pushed a new commit to fix this. Thanks.